### PR TITLE
Survey index page has pagination

### PR DESCRIPTION
**Pull request by aksh93**
Implementation for #25

### CODE REVIEW

File: rapidfire.gemspec [hunk 1]
```diff
@@ -22,6 +22,7 @@ PIM
 
   s.add_dependency 'rails', '>= 3.2.13'
   s.add_dependency 'active_model_serializers', '~> 0.8.1'
+  s.add_dependency 'kaminari'
```
Comment (iffyuva) [inline rapidfire.gemspec]: Can you make this a soft dependency? In code check if `Kaminari` is defined, and then do pagination.

File: spec/features/rapidfire/paginating_surveys_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe "Surveys with pagination" do
+  let!(:surveys) { FactoryGirl.create_list(:survey, 50) }
+  before do
+    visit rapidfire.root_path
+  end
+
+  it "lists all surveys within page scope" do
```
Comment (iffyuva) [inline spec/features/rapidfire/paginating_surveys_spec.rb]: merge this spec with index spec of surveys

File: rapidfire.gemspec (hunk 1 above)
Comment (aksh93) [reply rapidfire.gemspec]: Yeah, will do that.

**Comment by aksh93** [comment]
@iffyuva Had made the changes.

File: lib/rapidfire.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 require "rapidfire/engine"
+require "kaminari"
```
Comment (iffyuva) [inline lib/rapidfire.rb]: Can you try this PR with and without kaminari in a rails app?

Comment (aksh93) [reply lib/rapidfire.rb]: Yeah, it works out.

**Comment by aksh93** [comment]
@iffyuva removed the dependency from gemspec.

**Pull request merged**